notification-bot/src/index.ts

```diff
@@ -0,0 +1,215 @@
+import { StreamClient } from "@apibara/protocol";
+import {
+  Filter,
+  StarkNetCursor,
+  v1alpha2,
+  FieldElement,
+  EventFilter,
+  FilterBuilder,
+} from "@apibara/starknet";
+
+import { RpcProvider, constants, number, provider, uint256 } from "starknet";
+import { formatUnits } from "ethers";
+import * as dotenv from "dotenv";
+import { BlockNumber } from "starknet";
+import * as fs from "fs";
+import * as toml from "toml";
+import { Logger } from 'logger';
```

Comment:
Remove all the unused imports please, and also the dependencies that we don't use from package.json

Reply:
done